src/Model/Item.php

```diff
@@ -2292,9 +2292,9 @@ public static function expire(int $uid, int $days, string $network = "", bool $f
 				continue;
 			} elseif (!$expire_starred && intval($item['starred'])) {
 				continue;
-			} elseif (!$expire_notes && ($item['post-type'] == self::PT_PERSONAL_NOTE)) {
+			} elseif (!$expire_notes && (($item['post-type'] ?? null) == self::PT_PERSONAL_NOTE)) {
```

Comment:
Here we should have got a deeper look at the cause and not just fix the consequences.

Reply:
found the reason --> `post-type` wasn't at the "selected" list at `Post::select()`